ansible/playbooks/roles/compatibility_layer/templates/sync_gentoo_cache.j2

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+repository_name="${1}"
+repository_path="${3}"
+
+[[ ${repository_name} == "gentoo" ]] || exit 0
+
+source {{ gentoo_prefix_path }}/lib/gentoo/functions.sh
```

Comment:
Using the change below would allow you not to need templating this:
```suggestion
source $(portageq envvar EPREFIX)/lib/gentoo/functions.sh
```

Reply:
Thanks, that's much cleaner!